pkg/buildpack/multi_architecture_helper.go

```diff
@@ -39,11 +39,14 @@ func (m *MultiArchConfig) Targets() []dist.Target {
 // CopyConfigFiles will, given a base directory (which is expected to be the root folder of a single buildpack),
 // copy the buildpack.toml file from the base directory into the corresponding platform root folder for each target.
```

Comment:
Could you also update the docs to point out we also take care of extension.toml?

Reply:
Change is on this commi https://github.com/buildpacks/pack/pull/2320/commits/c002ca38f35e5d4634ca69c052a0f16a1aab6710